test/functional/p2p-segwit.py

```diff
@@ -271,6 +271,50 @@ def test_unnecessary_witness_before_segwit_activation(self):
         self.utxo.pop(0)
         self.utxo.append(UTXO(tx4.sha256, 0, tx4.vout[0].nValue))
 
+    # This test was added after segwit-activation:
+    # Version 0 witness outputs are unspendable before segwit activates
+    def test_v0_outputs_arent_spendable(self):
+        self.log.info("Testing that v0 witness program outputs aren't spendable pre-activation")
+
+        assert(len(self.utxo))
+
+        # Create two outputs, a p2wsh and p2sh-p2wsh
```

Comment:
comment says p2sh-p2wsh, but the second output below is just a p2sh as far as I can tell.

Reply:
strange. The P2SH output can't be spent either. If I change the `test_witness_block()` call to a `test_transaction_acceptance()` call below, I see the following log:

`(mandatory-script-verify-flag-failed (Operation not valid with the current stack size) (code 16))`

Reply:
Fixed -- I forgot to stick the p2sh redeem script in the scriptsig (and also inadvertently made this a p2sh, rather than p2sh-p2wsh).